### DISCUSSION TITLE
Deprecation of 2.9 for Win7 (which used to work on Win7)

### DISCUSSION THREAD

**Discussion by josephernest** [MiKTeX/miktex#1462] [General]
Hi,
Congrats to @edocevoli for this wonderful project!

I totally understand that new versions (Miktex 24.x) don't work with Win7, because this OS is end-of-life, and it makes no sense supporting Win7 for these new versions.

But why does MikTex 2.9 (which used to work perfectly on Win7) stops installing new packages on Win7? (I have seen this [announcement](https://miktex.org/announcement/legacy-windows-deprecation))
I have tried everything, including: 

1) setting

        set MIKTEX_MPM_REMOTESERVICE6100=multiplexor  
        set MIKTEX_MPM_REMOTESERVICE6210=multiplexor

2) creating a local filesystem package repo, and trying to install packages from local filesystem only (and no internet). But it still fails.

TL;DR **Would there be a a last-chance solution (ISO DVD with all CTAN packages?) for Windows 7?** (Including a paid solution which might be ok)

Thanks a lot in advance :)

PS: reason: I use a few computers, and some of them are on Win10, but some of them have to stay on Win7 because of legacy software (important for my activity) which don't work on Win10/11 or a Win7 VM.

PS2: there are lots of similar questions here, on StackOverflow, and elsewhere too. So a last chance solution for Win7 (maybe a 2020 CTAN  Miktex snapshot + 2.9 installer would be ok?) really makes sense for many people, who, for particular reasons can't upgrade all their computers to Win10 :)
